optacloud/src/main/resources/optacloud/optacloud/cloudScoreRules.drl

```diff
@@ -0,0 +1,63 @@
+package optacloud.optacloud;
+
+import org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScoreHolder;
+
+import optacloud.optacloud.Computer;
+import optacloud.optacloud.Process;
+
+global HardSoftScoreHolder scoreHolder;
+
+// ############################################################################
+// Hard constraints
+// ############################################################################
+
+rule "Hard CPU power capacity"
+    when
+        $computer : Computer($cpuPower : cpuPower)
+        $requiredCpuPowerTotal : Number(intValue > $cpuPower) from accumulate(
```

Comment:
This is the old accumulate style. In optaplanner-examples, I changed that some time ago:
  https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-examples/src/main/resources/org/optaplanner/examples/cloudbalancing/solver/cloudBalancingScoreRules.drl#L32
but I didn't adjust optacloud (I should have) - I think it's best to adjust it here too.